### PR TITLE
gdcache: fix gdcache test compile cmd

### PR DESCRIPTION
**Pull request by willson-chen**

### CODE REVIEW

File: src/gdcache.c [hunk 1]
```diff
@@ -25,7 +25,9 @@
  * John Ellson  (ellson@graphviz.org)  Oct 31, 1997
  *
  * Test this with:
- *               gcc -o gdcache -g -Wall -DTEST gdcache.c
+ *               gcc -o gdcache -g -Wall -DTEST -DNEED_CACHE gdcache.c -lgd
```
Comment (willson-chen) [inline src/gdcache.c]: libgd must be installed in this case

File: src/gdcache.c [hunk 2]
```diff
@@ -25,7 +25,9 @@
...
+ *               or
+ *               gcc -o gdcache -g -Wall -DTEST -DNEED_CACHE gdcache.c libgd.a
```
Comment (willson-chen) [inline src/gdcache.c]: we should specify the path of libgd.a in this case

Comment (pierrejoye) [reply src/gdcache.c]: Is it solved?

**Comment by willson-chen** [comment]
I don't know if someone will use the test code. But the compile cmd is incomplete.

**Pull request merged**